docs/hardware.rst

```diff
@@ -382,6 +383,36 @@ NUC before it can be used. We recommend:
 -  1x memory kit of compatible 2x8GB sticks
    -  You can put one 8GB memory stick in each of the servers.
 
+.. _nuc10_recommendation:
+
+Intel 10th-gen NUC
+~~~~~~~~~~~~~~~~~~
+We have tested and can recommend the `Intel NUC10i5FNH <https://ark.intel.com/content/www/us/en/ark/products/189239/intel-nuc-10-performance-kit-nuc10i5fnh.html>`__.
+It provides two storage options: M.2 SSD storage and a 2.5" secondary storage
+option (SSD or HDD).
+
+The NUC10's AX201 wireless hardware is not removable. Before installation of the
+RAM and storage, we recommend that you disconnect the wireless antennae leads
+from the AX201 component. They're the black and gray wires highlighted in
+the red box in the picture. Cover the free ends with electrical tape
+after disconnecting them.
+
+|NUC10 leads|
+
+Before the initial OS installation, boot into the BIOS by pressing **F2** at
+startup and adjust the system configuration:
+
+- Under **Advanced ▸ Onboard Devices**, disable all onboard devices
+  other than LAN: HD audio, microphone, Thunderbolt, WLAN, Bluetooth,
+  SD card controller, and enhanced consumer infrared.
```

Comment:
I don't know if the infrared option exists for older NUCs; it seems to be intended for remote controls, and worth enumerating.  The settings I have not enumerated here are:

- Gaussian Mixture Models and Neutral Networks Accelerator (enabled by default)
- HDMI CEC Control (enabled by default)
- Auto Turn on TV (set to "From S3/S4/S5 boot)
- Auto Turn OFF TV (disabled by default)
- Wake On TV (disabled by default)
- Standby by TV (disabled by default)

If any of those settings should be explicitly changed, I can enumerate them here as well.